src/screens/Home/surveyList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useRef } from 'react';
+
```

Comment:
minor thing 🚐  but the line break here is redundant IMO

Reply:
@namtx If you remember we used `import/order` of `eslint` to add the line breaks - https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m